entrypoint.sh

```diff
@@ -14,6 +14,7 @@ SRC_PATH="$INPUT_SRC_PATH"
 DST_PATH="$INPUT_DST_PATH"
 DST_OWNER="$INPUT_DST_OWNER"
 DST_REPO_NAME="$INPUT_DST_REPO_NAME"
+DST_REPO_DIR="$INPUT_DST_REPO_DIR"
```

Comment:
```suggestion
DST_REPO_DIR=dst_repo_dir
```
Move this variable/constant assignment down to the constants section. Around line 74.